q2_phylogeny/plugin_setup.py

```diff
@@ -41,13 +42,15 @@
 plugin.methods.register_function(
     function=q2_phylogeny.fasttree,
     inputs={'alignment': FeatureData[AlignedSequence]},
-    parameters={},
+    parameters={'n_threads': Int},
```

Comment:
I realize we didn't do this with mafft's `n_threads` parameter (it was an oversight on my part), but I think the following type makes better use of the framework's type system:

```
Int % Range(-1, None)
```